source/Nevermore.Tests/QueryBuilderFixture/QueryBuilderFixture.cs

```diff
@@ -28,7 +29,13 @@ public void SetUp()
         
         ITableSourceQueryBuilder<TDocument> CreateQueryBuilder<TDocument>(string tableName, string schemaName = "dbo", string idColumnName = "Id") where TDocument : class
         {
-            return new TableSourceQueryBuilder<TDocument>(tableName, schemaName, idColumnName, transaction, tableAliasGenerator, uniqueParameterNameGenerator, new CommandParameterValues(), new Parameters(), new ParameterDefaults());
+            var columns = new[] {"*"};
```

Comment:
in this situation, if the test is passing `CreateQueryBuilder<object>` it's okay to use `*` as the column names. The test is checking that it's building the query correctly, and by saying the column name is `*` it still shows the builder used the value.